src/config_validator.py

```
@@ -0,0 +1,156 @@
+    @classmethod
+    def load_from_env(cls) -> Dict[str, Any]:
+        """
+        Load configuration from environment variables.
+        
+        Returns:
+            Dict[str, Any]: Validated configuration from environment
+        """
+        config = {
+            'API_BASE_URL': os.getenv('COINGECKO_API_BASE_URL', 'https://api.coingecko.com/api/v3'),
+            'API_KEY': os.getenv('COINGECKO_API_KEY', ''),
+            'REQUEST_TIMEOUT': os.getenv('COINGECKO_REQUEST_TIMEOUT', 30),
```

Pylint (3.3.7)

[warning] os.getenv default type is builtins.int. Expected str or None.

(W1508)